lib/core/presentation/widgets/poap/hot_badge_item.dart

```diff
@@ -120,10 +159,23 @@ class _BadgeQuantityBarPainter extends CustomPainter {
 }
 
 class _BadgeLocationTag extends StatelessWidget {
-  const _BadgeLocationTag();
+  const _BadgeLocationTag({
+    required this.badge,
+  });
+
+  final badge_entities.Badge badge;
 
   @override
   Widget build(BuildContext context) {
+    final t = Translations.of(context);
+    final distanceInKm = (badge.distance ?? 0) / 1000;
+    String displayDistance;
```

Comment:
It's better have utility function just for getDisplayDistance, also can provide `prefix` or `suffix` for that

Reply:
:+1